DESCRIPTION

```diff
@@ -27,4 +23,4 @@ Encoding: UTF-8
 Language: en-US
 LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.1.2
+RoxygenNote: 7.2.0
```

Comment:
If we are updating here, perhaps we should also update in the template file as well

Reply:
The template repo is already updated to 7.2.0
https://github.com/insightsengineering/r.pkg.template/blob/main/DESCRIPTION#L30